pkg/systemd/services/service-details.jsx

```diff
@@ -494,16 +493,14 @@ export class ServiceDetails extends React.Component {
                                         {cockpit.format("$0 ($1)", this.props.unit.Listen[0][1], this.props.unit.Listen[0][0])}
                                     </DescriptionListDescription>
                                 </DescriptionListGroup>}
-                                <hr />
```

Comment:
This commit makes previous one about now showing this item obsolete.

Reply:
Removed.